helm/rustfs/templates/configmap.yaml

```diff
@@ -10,9 +10,9 @@ data:
   RUSTFS_LOG_LEVEL: {{ .Values.config.rustfs.log_level | quote }}
   {{- if .Values.mode.distributed.enabled }}
   {{- if eq (int .Values.replicaCount) 4 }}
-  RUSTFS_VOLUMES: "http://rustfs-{0...3}.rustfs-headless.rustfs.svc.cluster.local:9000/data/rustfs{0...3}"
+  RUSTFS_VOLUMES: "http://{{ include "rustfs.fullname" . }}-{0...3}.{{ include "rustfs.fullname" . }}-headless:9000/data/rustfs{0...3}"
```

Comment:
`http://{{ include "rustfs.fullname" . }}-{0...3}.{{ include "rustfs.fullname" . }}-headless.{{ include "rustfs.fullname" . }}.svc.cluster.local:9000/data/rustfs{0...3}`

The FQDN is better for k8s installation?